packages/layer-composer/src/types/index.ts

```diff
@@ -37,13 +37,22 @@ export enum Group {
  */
 export interface LayerMetadataLegend extends GeneratorLegend {
   id?: string
-  type: 'colorramp' | 'bivariate' | 'solid'
+  type: 'colorramp' | 'colorramp-discrete' | 'bivariate' | 'solid'
   gridArea?: number
```

Comment:
We have forgotten the `gridArea` property in heatmap-animated as it used to come from the stats, let's discuss if a key-value dictionary will be enough